devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -14,21 +14,28 @@
 # docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
 #
 ###
-FROM fabric8/java-alpine-openjdk8-jre:1.6.5
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-ENV AB_ENABLED=jmx_exporter
+ARG JAVA_PACKAGE=java-1.8.0-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.5
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install ${JAVA_PACKAGE} \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown -R 1001 /deployments \
+    && chmod -R "g+rwX" /deployments \
+    && chown -R 1001:root /deployments \
```

Comment:
you don't need any recursion as this directory was just created, no?

Reply:
Right, I took that from the existing file. We can simplify.

Reply:
Done.